core/src/main/resources/lib/form/helpLink.jelly

```diff
@@ -55,7 +55,11 @@ THE SOFTWARE.
     <j:when test="${attrs.url!=null}">
       <j:set var="altText" value="${attrs.featureName != null ? '%Help for feature:' + ' ' + attrs.featureName : '%[Help]'}" />
       <td class="setting-help">
-        <a href="#" class="help-button" helpURL="${rootURL}${attrs.url}"><l:icon class="icon-help icon-sm" alt="${altText}"/></a>
+        <a href="#" class="help-button" helpURL="${rootURL}${attrs.url}">
+          <l:svgIcon class="icon-help"
+                     tooltip="${altText}"
+                     href="${resURL}/images/material-icons/svg-sprite-action-symbol.svg#ic_help_24px" />
```

Comment:
Can we do this more generically in the tag perhaps? This looks like it would actively hinder another iteration of icons given the amount of implementation detail referenced everywhere we put an icon.

Reply:
done